app/src/main/res/xml/settings.xml

```diff
@@ -24,6 +24,14 @@
         android:defaultValue="@string/settings_dark_default"
         android:key="@string/settings_dark_key"
         android:title="@string/settings_dark_label"/>
+    <EditTextPreference
+        android:defaultValue="0"
```

Comment:
Here you should use a reference to the default value you specified in the strings.xml file.